ext/dcos-installer/dcos_installer/installer_analytics.py

```diff
@@ -20,14 +20,14 @@ def send(self, action, install_method, num_errors):
         # We set customer key from config to avoid loading the config during class init
         customer_key = backend.get_config().get("customer_key", None)
 
-        analytics.track(user_id=customer_key, anonymous_id=self.uuid, event=action, properties={
+        analytics.track(user_id=customer_key, anonymous_id=self.uuid, event="installer", properties={
             "provider": "onprem",
             "source": "installer",
-            "variant": os.environ["BOOTSTRAP_VARIANT"],
+            "variant": 'open' if os.environ["BOOTSTRAP_VARIANT"] == '' else 'enterprise',
```

Comment:
This is not ok. We can't have enterprise implementation bleed into the open project.